### DISCUSSION TITLE
Minimal mode?

### DISCUSSION THREAD

**Discussion by devdumpling** [dailydotdev/daily#1343] [Feature Requests]
Howdy! 

I just joined after hearing about y'all on the [Syntax.fm podcast](https://syntax.fm/show/809/how-to-stay-up-to-date-with-daily-dev-s-francesco-ciulla) (thanks Francesco!). 

Decided to give it a shot after listening and my initial impressions were:

1. Impressed by the quality and relevance of deep (not shallow clickbait articles) content 
2. I'm completely overwhelmed by content.

It's the latter point that I'm here to discuss. Y'all have this handy feedback thing, so...

**PREFACE EDIT: I discovered list view thanks to a comment below, as well as the density options. These get me closer to what I'm looking for.**

This is what my initial feed looks like:

<img width="2346" alt="Screenshot 2024-08-17 at 2 55 17 PM" src="https://github.com/user-attachments/assets/3f8fe97b-33a4-44b7-9ee5-b00037f7de53">

Very busy. There's... a lot to see here. Many cards, many images, overall too much content for me. I'm sure many people prefer this, but the density of information (unexpectedly) turns me off from wanting to read through each one of these. 

> Now one caveat, I'm viewing on a larger screen (27" studio display with some high DPI scaling)--this is not so bad on just a normal laptop display I'm sure. 

## The Feature

I would love a "minimal" landing page mode, where I could simply toggle off certain things and maybe get a little more whitespace. Honestly even just hiding the thumbnails and moving from a card grid to a simple list is probably enough. I can probably hack this together with [Arc](https://arc.net/) to override styles, but it would be cool to have a built-in. 

Not a designer, but here'd be my ideal minimal mode:

- Move from cards to a list, where each list item includes author avatar, title, upvotes, bookmark, link, and the expandy dots (the comment button seems mostly redundant with clicking on the item, and the tags are nice for filtering but don't give me much more than the title does personally)
- Remove thumbnail or make it smaller (personal preference as I see myself using daily.dev more to scan for news). 
- Make the content modal larger on large screens (again, first-world-27in-monitor-problems)

<img width="1181" alt="Screenshot 2024-08-17 at 3 10 48 PM" src="https://github.com/user-attachments/assets/0689fc2a-8a92-4ea9-9e49-2f2d36431d07">

> Would love this modal to be a touch wider 

At the risk of "Redditifying" too much, I'm a less is more person, and would definitely use a more "minimal theme" if it were available. 

----

Thanks for all the work y'all do. As a new user, very impressed 🎉

**Comment by plplmax** [comment]
Btw, you can change layout to the list by clicking on your avatar at the top right corner and click "Customize"

**Reply by devdumpling** [reply]
Oh! And there's also card density! Shame on me for not looking harder 😅

**Reply by idoshamun** [reply]
Shame on us for not making it easily accessible 😉

**Comment by idoshamun** [comment]
@devdumpling thanks a lot for this feedback. We will consider what's the best way to address it
